package.json

```diff
@@ -172,6 +171,7 @@
         "@typescript-eslint/parser": "8.32.1",
         "@vercel/nft": "0.29.3",
         "@vitest/coverage-v8": "2.1.9",
+        "cross-env": "^7.0.3",
```

Comment:
`cross-env` is indeed a `dependencies`. Moving it to `devDependencies` will make RSSHub not able to start https://github.com/DIYgod/RSSHub/blob/59e7ea876df12bbaf8a4f72d5de12f99f8f9cff3/package.json#L34
Are there any strong reasons to move it to `devDependencies`?

Also why change the version range to caret? Every dependency is expected to be pinned to its exact version.